account_tax_analysis/views/account_move_line.xml

```diff
@@ -0,0 +1,88 @@
+
+<odoo>
+    <record id="view_account_move_line_filter_vat_analysis" model="ir.ui.view">
+        <field name="name">Journal Items Tax</field>
+        <field name="model">account.move.line</field>
+        <field eval="32" name="priority"/>
+        <field name="arch" type="xml">
+            <search>
+                <field name="date"/>
+                <field name="account_id"/>
+                <group name="filters">
+                    <filter
+                          string="Unposted"
+                          domain="[('move_id.state','=','draft')]"
+                          help="Unposted Journal Items"/>
+                    <filter name="posted"
+                          string="Posted"
+                          domain="[('move_id.state','=','posted')]"
+                          help="Posted Journal Items"/>
+                </group>
+                <group name="group_by">
+                    <filter string="Partner"
+                          name="group_by_partner"
+                          domain="[]"
+                          context="{'group_by': 'partner_id'}"/>
+                    <filter string="Account"
+                          name="group_by_account"
+                          context="{'group_by': 'account_id'}"/>
+                    <filter name="group_by_month"
+                          string="Month"
+                          context="{'group_by': 'date:month'}" />
+                    <filter name="group_by_year"
+                          string="Year"
+                          context="{'group_by': 'date:year'}" />
+                    <filter string="Journal"
+                          name="group_by_journal"
+                          domain="[]"
+                          context="{'group_by': 'journal_id'}"/>
+                    <filter string="Tax / Account"
+                          name="group_by_account_tax"
+                          domain="[]"
+                          context="{'group_by': ['analysis_tax', 'account_id'] }"/>
+                    <filter string="Tax account"
+                          name="group_by_tax"
+                          domain="[]"
+                          context="{'group_by': 'analysis_tax'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="view_move_line_tree_tax_analysis" model="ir.ui.view">
+        <field name="name">account.move.line.tree</field>
+        <field name="model">account.move.line</field>
+        <field eval="16" name="priority"/>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <field name="analysis_tax" />
+                <field name="name"/>
+                <field name="ref"/>
+                <field name="date"/>
+                <field name="partner_id" />
+                <field name="account_id" domain="[('journal_id','=',journal_id), ('company_id', '=', company_id)]"/>
```

Comment:
what's the purpose of this domain?

Reply:
I kept it from 8.0, I suppose it's for the editable="top". Now that you point it, journal_id and company_id isn't in the tree, so it will failed.

Note that when the tree is grouped, which is set by default, the editable don't work and the form is showed.
Is it useful to keep the editable? If yes, I will add the fields.

Reply:
I see no reason to have this view editable. You can disable create too by the way.